taxi/form.py

```diff
@@ -0,0 +1,21 @@
+from django import forms
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver
+
+
+class DriverForm(forms.ModelForm):

```

Comment:
you have to create 2 forms: for creating the driver (must inherit UserCreationForm) and for updating license_number. In both forms, you must validate license_number. And DriverCreationForm should have only username, password, email, first_name, last_name and license_number fields